osquery/tables/system/posix/smbios_utils.cpp

```diff
@@ -187,6 +187,9 @@ const std::map<uint8_t, std::string> kSMBIOSMemoryErrorOperationTable = {
     {0x05, "Partial write"},
 };
 
+const std::map<std::string, std::string> kSMBIOSProcessorTypeFriendlyName = {
+    {"3", "CPU"}, {"4", "MATH"}, {"5", "DSP"}, {"6", "GPU"}};
+
```

Comment:
Is it reasonable to comment where these came from?

Reply:
I've added a comment on the top of the various mappings, since all the mappings are taken from the SMBIOS spec PDF.